tests/filecheck/dialects/emitc/emitc_ops.mlir

```diff
@@ -0,0 +1,24 @@
+// RUN: XDSL_ROUNDTRIP
+// RUN: XDSL_GENERIC_ROUNDTRIP
+
+//===----------------------------------------------------------------------===//
+// CallOpaqueOp
+//===----------------------------------------------------------------------===//
+
+%0 = emitc.call_opaque "blah"() : () -> i64
+emitc.call_opaque "foo" (%0) {args = [
+  0 : index, dense<[0, 1]> : tensor<2xi32>, 0 : index
+]} : (i64) -> ()
+emitc.call_opaque "test" ()  : () -> ()
+
+// CHECK: builtin.module {
+// CHECK-NEXT:  %0 = emitc.call_opaque "blah"() : () -> i64
```

Comment:
The mlir docs seem to suggest there should be space between "blah" and (), would it be possible to check if this agrees with `mlir-opt`?